src/sentry/relocation/tasks/process.py

```diff
@@ -1380,6 +1451,12 @@ def validating_complete(uuid: UUID, build_id: str) -> None:
     acks_late=True,
     soft_time_limit=SLOW_TIME_LIMIT,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of SLOW_TIME_LIMIT to align with the celery configuration?